src/app/components/Item/ItemHoldings.jsx

```diff
@@ -49,9 +49,10 @@ class ItemHoldings extends React.Component {
       page: noItemPage ? 1 : this.state.page,
     });
 
-    if (noItemPage) {
-      this.context.router.replace(`${appConfig.baseUrl}/bib/${this.props.bibId}`);
-    }
+    // if (noItemPage) {
+    //   console.log('replacing')
+    //   this.context.router.replace(`${appConfig.baseUrl}/bib/${this.props.bibId}`);
+    // }
```

Comment:
This is a redirect to the same bib page when the item page in the url is bigger than the number of pages available. Here's an example: https://www.nypl.org/research/collections/shared-collection-catalog/bib/b12404206

This bib has 46 items and 3 "pages" for the items. If you click on "Next", the url changes to `research/collections/shared-collection-catalog/bib/b12404206?itemPage=2`. If you share this url, you'll land on the second page of items for this bib. Likewise, if you directly change the url to `itemPage=3`, this works. But if you change this to `itemPage=4`, this shouldn't work! So this simply redirects to the main bib page.

This was more of a fail-safe in case any user updated the url directly, not that it's expected but good to catch user behavior.